packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.js

```diff
@@ -91,7 +91,7 @@ class Dropdown extends React.Component {
             isOpen={isOpen}
             position={position}
             aria-labelledby={id}
-            onClick={event => onSelect && onSelect(event)}
+            onSelect={event => onSelect && onSelect(event)}
```

Comment:
Actually, now that I think about it again, I realize that `<Dropdown />` isn't actually a select... it's a menu.  So `onChange` doesn't necessarily make sense either.

This is sort of a weird use case here...  What do you think we should do?

Reply:
This may be more of a case of bad nomenclature, as this onSelect isn't a property of the DropdownMenu but a prop that gets passed down to the DropdownItem itself where it is set to the onClick function.

Reply:
```suggestion
             onSelect={onSelect}
```